docs/content/preview/yugabyte-cloud/cloud-secure-clusters/managed-ear.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Encryption at rest
+linkTitle: Encryption at rest
+description: YugabyteDB Managed cluster encryption at rest.
+headcontent: Encrypt your YugabyteDB cluster
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-ear
+    parent: cloud-secure-clusters
+    weight: 460
+type: docs
+---
+
+YugabyteDB Managed uses volume encryption for all data at rest, including your account data and your clusters and their backups. Data is AES-256 encrypted using native cloud provider technologies - S3 and EBS volume encryption for AWS, and server-side and persistent disk encryption for GCP. Volume encryption keys are managed by the cloud provider and anchored by hardware security appliances.
+
+In addition to cloud provider volume encryption, you can enable YugabyteDB encryption at rest (EAR) for clusters. When enabled, your cluster (including backups) is encrypted using a customer managed key (CMK) residing in a cloud provider Key Management Service (KMS). (Currently, only AWS KMS is supported.) You can grant YugabyteDB Managed access to the key with the requisite permissions to perform cryptographic operations using the key to secure the databases in your clusters.
+
+You enable YugabyteDB EAR for a cluster when you create it.
+
+<!-- You can enable YugabyteDB EAR for a cluster as follows:
+
+- On the **Security** page of the **Create Cluster** wizard when you create a cluster.
+- On the cluster **Settings** tab under **Encryption at rest**.
+-->
+You must be signed in as an Admin user to enable cluster EAR.
+
+## Limitations
+
+- Currently, EAR is only supported for AWS KMS.
+- You cannot remove encryption from clusters that have EAR enabled.
+- After EAR is enabled for a cluster, you cannot change keys.
+
+## Prerequisites
+
+### AWS
+
+- Single-region [symmetric encryption key](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#symmetric-cmks) created in AWS KMS. The key resource should have the following permissions:
+  - kms:Encrypt
+  - kms:Decrypt
+  - kms:GenerateDataKeyWithoutPlaintext
+  - kms:DescribeKey
+  - kms:ListAliases
+- Amazon Resource Name (ARN) of the CMK. For more information, refer to [Amazon Resource Names](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) in the AWS documentation.
+- An access key for an [IAM identity](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html) with permission to encrypt and decrypt using the CMK. An access key consists of an access key ID and the secret access key. For more information, refer to [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) in the AWS documentation.
```

Comment:
Can I have multiple IAM identities, meaning multiples users/roles for a single cluster and enable encryption for each identity?